client/src/components/layout/ReviewsList.js

```diff
@@ -1,10 +1,11 @@
 import React from "react"
 import ReviewTile from "./ReviewTile"
 
-const ReviewsList = ({reviews}) => {
+const ReviewsList = (props) => {
+    const reviews = props.reviews
     const listOfReviews = reviews.map((review) => {
         return (
-            <ReviewTile key={reviews.id} review={review}/>
+            <li key={review.id}><ReviewTile review={review}/></li>
```

Comment:
nope, you should not be wrapping an `li` tag around this Review. If every review needs to be listed in an li, put the `li` IN the `ReviewTile`